Sources/Commands/SwiftTool.swift

```diff
@@ -472,6 +473,17 @@ public class SwiftTool<Options: ToolOptions> {
         binder.bind(
             option: parser.add(option: "--build-system", kind: BuildSystemKind.self, usage: nil),
             to: { $0._buildSystem = $1 })
+        
+        #if os(macOS)
+        if #available(macOS 10.13, *) {
+            // implementation of the netrc reader depends upon macOS 10.13
+            binder.bind(
+                option: parser.add(
+                    option: "--netrc-file", kind: PathArgument.self,
+                    usage: "Absolute path to netrc file for downloading binary target artifacts"),
+                to: { $0.netrcFilePath = $1.path })
+        }
+        #endif
```

Comment:
I think we should have the option always be present and emit an error if it is being used on non-macOS. It doesn't really matter right now since binaries aren't supported on non-macOS, but I think silently ignored options can be confusing.

Reply:
Added check to post-processing block
https://github.com/apple/swift-package-manager/pull/2833/files#diff-b1d92ba41a0ed8ddabdf8c2604a41504R572-R584